assets/js/accordion.js

```diff
@@ -0,0 +1,33 @@
+
+window.onload = function () {
+    const accordionHeaders = document.querySelectorAll('.accordion-heading');
+    // const expand = document.querySelector('.expand-all-accordion .expand');
+    // const close = document.querySelector('.close-all-accordion .close');
+    console.log(accordionHeaders);
```

Comment:
Looks like we have a few `console.log` that could be good to remove once it is working.